src/avfoundation.cs

```diff
@@ -2573,6 +2573,9 @@ interface AVAsset : NSCopying {
 		NSString WasDefragmentedNotification { get; }
 #endif
 
+		[iOS (10, 2), Mac (10,12,2), TV (10, 2)]
```

Comment:
No worries, but in the diff I see NS_AVAILABLE(10_12_3, 10_3) and in the Apple docs the supported platforms are:
* iOS 10.3+
* macOS 10.12.3+
* tvOS 10.3+

So I'm more inclined in changing the Mac attribute than the TV one, what do you think?